py/modmicropython.c

```diff
@@ -200,6 +201,9 @@ static const mp_rom_map_elem_t mp_module_micropython_globals_table[] = {
     #if MICROPY_KBD_EXCEPTION
     { MP_ROM_QSTR(MP_QSTR_kbd_intr), MP_ROM_PTR(&mp_micropython_kbd_intr_obj) },
     #endif
+    #if MICROPY_PY_MICROPYTHON_RINGIO
+    { MP_ROM_QSTR(MP_QSTR_RingIO), MP_ROM_PTR(&mp_type_micropython_ringio) },
```

Comment:
I suggest this go at the end of the locals dict, classes come after functions (generally).

Reply:
Ah, I was going with basic alphabetical there! How about on the docs page? Should it be moved similarly?

Reply:
It should also be moved in the docs.  You'll see there's an existing "Functions" heading, so please add a "Classes" heading at the end of the file, then RingIO docs.

Reply:
It looks like `RingIO` is still not at the end of the locals dict here?